cloudnatix/manifest.json

```diff
@@ -0,0 +1,71 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "caebfbe2-48ba-443f-b2c6-bd80122b2605",
+  "app_id": "cloudnatix",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Integration with CloudNatix",
```

Comment:
For the description, can you instead add detail about what CloudNatix does? i.e:

Provides automated capacity, cost & operation optimization from CloudNatix

Reply:
updated.